apps/dolly-frontend/src/main/js/src/pages/adminPages/dollyInfostriper/DollyInfostripePage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useDollyInfostriper } from '@/utils/hooks/useDollyInfostriper'
+import { CreateInfostripeForm } from '@/pages/adminPages/dollyInfostriper/CreateInfostripeForm'
+import { Alert, Box } from '@navikt/ds-react'
+import Loading from '@/components/ui/loading/Loading'
+import { InfostripeListItem } from '@/pages/adminPages/dollyInfostriper/InfostripeListItem'
+import { TestComponentSelectors } from '#/mocks/Selectors'
+
+export default function DollyInfostripePage() {
+	const { infostriper, loading, error } = useDollyInfostriper()
+
+	return (
+		<>
+			<h1>Infostripe-oversikt</h1>
+			<p>Her finner du en oversikt over alle eksisterende infostriper som vises i Dolly.</p>
+			<CreateInfostripeForm />
+			{loading && (
+				<div data-testid={TestComponentSelectors.LOADING_INFOSTRIPER}>
+					<Loading label="Laster infostriper ..." />
+				</div>
+			)}
+			{error && <Alert variant="warning">{`Feil ved henting av data: ${error.message}`}</Alert>}
+			{infostriper?.length > 0 && (
```

Comment:
Det blir en del ulike elementer under hverandre på denne siden - kunne det vært fint å dele opp litt? Siden vi har en mellomtittel på opprett ny infostripe, ville det vært nice å hatt en over infostripe-lista også, f.eks. "Aktive infostriper" eller noe? Og kaaanskje det hjelper med en linje over hver av disse mellomtitlene, så form kommer for seg, og lista kommer for seg?